### PR TITLE
[Chore: #162727503] Configure Travis and Coveralls

### PR DESCRIPTION
**Pull request by ebenezerdon**
#### What does this PR do?
This PR configures Travis and Coveralls for the project

#### Description of Task to be completed?
- This story should include the basic setup of jest and Travis.
- A simple test for one of the components should be added.

#### How should this be manually tested?
- When a branch remote branch is pushed, Travis should run tests for that branch
- run `dev:test` to see the current test result for the `LandingPage component test`

### CODE REVIEW

File: webpack/webpack.dev.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
- const common = require('./webpack.common.js');
+/* eslint-disable import/no-extraneous-dependencies */
+const merge = require('webpack-merge');
+const common = require('./webpack.common.js');
 
- module.exports = {
-   ...common,
-   mode: 'development'
-  };
+module.exports = merge(common, {
```
Comment (andela-mabdussalam) [inline webpack/webpack.dev.js]: Why did you change back to `webpack-merge`?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - "stable"
+script:
+  - npm test
```
Comment (andela-mabdussalam) [inline .travis.yml]: Add a new line.

File: webpack/webpack.dev.js (hunk 1 above)
Comment (ebenezerdon) [reply webpack/webpack.dev.js]: This resulted from a merge conflict. I've fixed it.

**Review by andela-mabdussalam — APPROVED** [review]
